SSD/ssd_cmd_buffer_control_test.cpp

```diff
@@ -346,5 +397,151 @@ TEST_F(BufferControlFixture, BufferWrite2) {
   argv[4] = nullptr;
   cmdBuffer.runCommandBuffer(argv);
 
+  EXPECT_EQ(expected_ret, cmdBuffer.getBufferNameList());
+}
+
+TEST_F(BufferControlFixture, BufferReadWithRunCommandPass) {
+
+  EXPECT_TRUE(cmdBuffer.updateToNextEmpty("W_19_0x11112222"));
+
+  char *argv[5];
+  argv[0] = const_cast<char *>("ssd.exe");
+  argv[1] = const_cast<char *>("R");
```

Comment:
argv[0] 가 SSD.exe 아닌가요?

```suggestion
  char *argv[5];
  argv[0] = const_cast<char *>("SSD.exe");
  argv[1] = const_cast<char *>("R");
```

Reply:
말씀해주신 부분 수정했습니다.